src/directives/pagination.ts

```diff
@@ -40,7 +40,7 @@ export interface PaginationDirectiveConfiguration<T> {
 export const paginationDirective = <T>({table}: PaginationDirectiveConfiguration<T>): PaginationDirective => {
     let {slice: {page: currentPage, size: currentSize}} = table.getTableState();
     let itemListLength = table.filteredCount;
```

Comment:
In what situation would you have falsy filteredCount ? 
I would prefer in that case to set a default value here ``let itemListLength = table.filteredCount || 0;``

Reply:
ha my bad the result would be correct

Reply:
Wouldn't filteredCount be 0 if there are no results after filtering the table?